### DISCUSSION TITLE
ListOperation with sub-records

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#439] [Q&A (Help)] [answered]
I remember some time ago seeing in the demo a setupListOperation type display in which each record had a small + symbol on the left, clicking on which you could see the connected records, such as invoices and invoice lines. How do you set up a list like this?

Thanks
Mauro

**Comment by promatik** [comment, marked as answer]
Hi Mauro!

1) **Details Row**
That `[+]` button is the details row, you can read more about it in the documentation: https://backpackforlaravel.com/docs/5.x/crud-operation-list-entries#details-row-1
You can place there anything you want, it's up to you to create the view with anything you want 👌

2) **Relation Tables**
If you want to have a table with the connected records (relationships), you have an addon from @izica that does exactly that;
https://github.com/izica/relations-widgets-for-backpack
You can place many of this widgets inside the details row 👌

**Reply by blondie63** [reply]
About N. 2 , is there some online demo ? because their screenshots are all in russian..

**Reply by tabacitu** [reply]
No online demo, sorry. All info about that is in that 3rd party package.
